.github/workflows/push.yaml

```diff
@@ -6,6 +12,13 @@ on:
   pull_request:
   workflow_dispatch:
 jobs:
+  check-licenses:
+    name: Check licenses
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: REUSE Compliance Check
+      uses: fsfe/reuse-action@v2
```

Comment:
I'd prefer these to use `seL4/ci-actions/license-check@master` instead so that there is only one global setup and configuration for the license check. The seL4 repo at

https://github.com/seL4/seL4/blob/7221eba1b96c3c3b508345fee9a564c32375da0b/.github/workflows/push.yml#L14-L19

has an example of how it is used.

Reply:
Fixed